### PR TITLE
Implemented affine cipher encryption and decryption. 

### PR DESCRIPTION
**Pull request by tvuolio**
#14 The class methods in MethodFactory were also updated accordingly.

### CODE REVIEW

File: JEasyCrypto/src/easycrypto/AffineCipherMethod.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package easycrypto;
+
+import easycrypto.EasyCryptoAPI.Result;
+import easycrypto.EasyCryptoAPI.ResultCode;
+import java.lang.Math;
```
[the remark this answers is not in the document]
Comment (tvuolio) [reply JEasyCrypto/src/easycrypto/AffineCipherMethod.java]: You are absolutely correct. We first used the "abs"-function, but decided to use a different implementation to carry out the same job. The import will be removed from the PR.

File: JEasyCrypto/src/easycrypto/AffineCipherMethod.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+class AffineCipherMethod implements CryptoMethod {
```
[the remark this answers is not in the document]
Comment (tvuolio) [reply JEasyCrypto/src/easycrypto/AffineCipherMethod.java]: We will change the formatting of the comments into the requested form.

**Comment by tvuolio** [comment]
The changes requested have now been made.

**Comment by mbuet2ner** [comment]
Thank you @tvuolio for your work! Looks really good. I will merge it!

**Pull request merged**